vaultenv.cabal

```diff
@@ -0,0 +1,87 @@
+cabal-version: 2.3
+name: vaultenv
```

Comment:
This change was intentional. We changed this to `vaultenv-real` because of the interaction with the `vaultenv` from Nixpkgs in `static-haskell-nix`. That's no longer a problem.